tests/pds/dataservice_test.go

```diff
@@ -337,6 +340,24 @@ var _ = Describe("{ScaleUPDataServices}", func() {
 						pod, dep, err = pdslib.CreateDataServiceWorkloads(ds.Name, deployment.GetId(), "", "", deploymentName, namespace)
 						log.FailOnError(err, "Error while genearating workloads")
 					}
+					if ds.Name == elasticSearch {
+						deploymentName := "es-rally"
+						log.Infof("Running Workloads on DataService %v ", ds.Name)
+						pod, dep, err = pdslib.CreateDataServiceWorkloads(ds.Name, deployment.GetId(), "", "", deploymentName, namespace)
+						log.FailOnError(err, "Error while genearating workloads")
+					}
+					if ds.Name == elasticSearch {
```

Comment:
Is there a reason we are running Elasticsearch twice ?

Reply:
It was added by mistake and its a good catch, Thanks and I have removed it.